pom.xml

```diff
@@ -18,46 +18,53 @@
         <skip.integration.tests>false</skip.integration.tests>
         <skip.unit.tests>false</skip.unit.tests>
 
-        <java.version>1.8</java.version>
-        <java-session-handler.version>2.2.0</java-session-handler.version>
-        <private-api-sdk-java.version>2.0.325</private-api-sdk-java.version>
-        <sdk-manager-java.version>1.5.1</sdk-manager-java.version>
-        <api-helper-java.version>1.1.0-rc1</api-helper-java.version>
-        <rest-service-common-library.version>0.0.3</rest-service-common-library.version>
-        <environment-reader-library.version>1.3.2</environment-reader-library.version>
-        <jackson.version>2.15.3</jackson.version>
+        <java.version>21</java.version>
+        <java-session-handler.version>4.0.7</java-session-handler.version>
+        <private-api-sdk-java.version>4.0.142</private-api-sdk-java.version>
+        <sdk-manager-java.version>3.0.5</sdk-manager-java.version>
+        <api-helper-java.version>3.0.1</api-helper-java.version>
+        <rest-service-common-library.version>2.0.2</rest-service-common-library.version>
+        <environment-reader-library.version>3.0.1</environment-reader-library.version>
+        <jackson.version>2.17.1</jackson.version>
         <!-- Logging -->
-        <log4j-bom.version>2.17.0</log4j-bom.version>
-        <structured-logging.version>1.9.12</structured-logging.version>
+        <log4j-bom.version>3.0.0-alpha1</log4j-bom.version>
+        <structured-logging.version>3.0.8</structured-logging.version>
         <avro.version>1.11.3</avro.version>
-        <ch-kafka.version>1.4.4</ch-kafka.version>
-        <kafka-models.version>1.0.12</kafka-models.version>
-        <ojdbc6.version>11.2.0.3</ojdbc6.version>
+        <ch-kafka.version>3.0.2</ch-kafka.version>
+        <kafka-clients.version>3.7.0</kafka-clients.version>
+        <ojdbc11.version>23.4.0.24.05</ojdbc11.version>
         <junit-jupiter.version>5.7.1</junit-jupiter.version>
         <mockito-junit-jupiter.version>3.8.0</mockito-junit-jupiter.version>
```

Comment:
Latest version of mockito junit jupiter is [5.12.0](https://mvnrepository.com/artifact/org.mockito/mockito-junit-jupiter/5.12.0)

Reply:
mockito-inline latest version is 5.2.0 - updated